src/exoscale/tools/project/api/jar.clj

```diff
@@ -1,8 +1,12 @@
 (ns exoscale.tools.project.api.jar
   (:require [clojure.tools.build.api :as b]
-            [exoscale.tools.project.api :as api]
             [exoscale.tools.project.api.version :as v]
             [exoscale.tools.project.dir :as dir]))
+(defn- create-basis
```

Comment:
I would leave create-basis in a more "shared" namespace, it has more uses than just creating (uber)jars.

Reply:
it's only used here for now, it can move when we need it elsewhere